### PR TITLE
Update sanidhya.json

### PR DESCRIPTION
**Pull request by codersonet**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

**Review by wdhdev — CHANGES_REQUESTED** [review]
No CNAME with other records

**Comment by CoderSoNet** [comment]
> 

What does it mean?

File: domains/sanidhya.json [hunk 1]
```diff
@@ -3,17 +3,20 @@
...
     "discord": "781127625263415326"
   },
   "record": {
-    "A": [
-      "185.199.109.153",
-      "185.199.110.153",
-      "185.199.111.153",
-      "185.199.108.153"
-    ],
+
```
Comment (VaibhavSys) [inline domains/sanidhya.json:11]: You should remove the extra blank lines

**Review by wdhdev — APPROVED** [review]

File: domains/sanidhya.json (hunk 1 above)
Comment (WilliamDavidHarrison) [reply domains/sanidhya.json]: Done.

**Pull request merged**